src/nnet2/nnet-component.h

```diff
@@ -1784,6 +1784,7 @@ class Convolutional1dComponent: public UpdatableComponent {
   const Convolutional1dComponent &operator = (const Convolutional1dComponent &other); // Disallow.
   CuMatrix<BaseFloat> filter_params_;
   CuVector<BaseFloat> bias_params_;
+  bool appended_conv_;
```

Comment:
Yes, but if someone looks for that documentation they would never find it--
at least you need to state by the declaration of the variable, where
someone would find the documentation.

On Tue, Oct 27, 2015 at 11:08 PM, Xingyu Na <notifications@github.com>
wrote:

> In src/nnet2/nnet-component.h
> <https://github.com/kaldi-asr/kaldi/pull/296#discussion_r43214509>:
>
> > @@ -1784,6 +1784,7 @@ class Convolutional1dComponent: public UpdatableComponent {
> >    const Convolutional1dComponent &operator = (const Convolutional1dComponent &other); // Disallow.
> >    CuMatrix<BaseFloat> filter_params_;
> >    CuVector<BaseFloat> bias_params_;
> > +  bool appended_conv_;
>
> Do you mean the used of appended_conv_? It is documented at Propogate().
> Xingyu
> … <#150ac6b023d5a91e_>
> On 10/28/2015 10:55 AM, Daniel Povey wrote: In src/nnet2/nnet-component.h <#296
> (comment)
> <https://github.com/kaldi-asr/kaldi/pull/296#discussion_r43214030>>: > @@
> -1784,6 +1784,7 @@ class Convolutional1dComponent: public
> UpdatableComponent { > const Convolutional1dComponent &operator = (const
> Convolutional1dComponent &other); // Disallow. > CuMatrix<BaseFloat>
> filter_params_; > CuVector<BaseFloat> bias_params_; > + bool
> appended_conv_; This should be documented with a comment at this point. —
> Reply to this email directly or view it on GitHub <
> https://github.com/kaldi-asr/kaldi/pull/296/files#r43214030>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/296/files#r43214509>.
>